specs/deneb/fork-choice.md

```diff
@@ -27,37 +27,36 @@ This is the modification of the fork choice accompanying the Deneb upgrade.
 #### `validate_blob_sidecars`
 
 ```python
-def validate_blob_sidecars(slot: Slot,
-                           beacon_block_root: Root,
-                           expected_kzg_commitments: Sequence[KZGCommitment],
-                           blob_sidecars: Sequence[BlobSidecar]) -> None:
-    assert slot == blobs_sidecar.beacon_block_slot
-    assert beacon_block_root == blobs_sidecar.beacon_block_root
-    assert len(expected_kzg_commitments) == len(blob_sidecars)
-    # TODO validate commitments individually or aggregate first?
-    # assert verify_aggregate_kzg_proof(blobs, expected_kzg_commitments, kzg_aggregated_proof)
+def validate_blobs(expected_kzg_commitments: Sequence[KZGCommitment],
```

Comment:
Slight preference for `blobs` to be first argument given it is what the function claims to validate.

edit: I see it maps to the order of `verify_blob_kzg_proof()`. so probably fine to just leave the parallelism

Reply:
if you go on a crypto parameter reordering crusade, I'll follow :stuck_out_tongue: